src/calculation/ClauseResultsBuilder.ts

```diff
@@ -757,7 +769,11 @@ export function buildPopulationRelevanceMap(
   }
 
   // If NUMER is true then DENEXCEP is not calculated
-  if (hasResult(PopulationType.NUMER, results) && getResult(PopulationType.NUMER, results) === true) {
+  if (
+    hasResult(PopulationType.NUMER, results) &&
+    getResult(PopulationType.NUMER, results) === true &&
+    !MeasureBundleHelpers.isRatioMeasure(group, measureScoringCode)
```

Comment:
My only suggestion would be to maybe add a comment there about the ratio measure stuff like you did above (line 750) but that's pretty picky haha